src/prefect/orion/schemas/schedules.py

```diff
@@ -394,6 +396,11 @@ def validate_rrule_str(cls, v):
             # rrules errors are a mix of cryptic and informative
             # so reraise to be clear that the string was invalid
             raise ValueError(f'Invalid RRule string "{v}": {exc}')
+        if len(v) > MAX_RRULE_LENGTH:
+            raise ValueError(
+                f'Invalid RRule string "{v}"\n'
```

Comment:
We probably don't want to print 6500 characters in the value error. Let's truncate it and add a trailing "..."